buildSrc/src/main/kotlin/hyper-conventions.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 val hyperApiVersion: String by project
-val hyperZipName = "hyper-$hyperApiVersion.zip"
+val hypzerZipPath = ".hyper/hyper-$hyperApiVersion.zip"
```

Comment:
Sneak in a tiny tweak to not put the hyper zip in `build`, which would blow it away every time you `gradle clean`